plugins/uefi-capsule/fu-plugin-uefi-capsule.c

```diff
@@ -366,6 +366,50 @@ fu_plugin_uefi_capsule_update_splash (FuPlugin *plugin, FuDevice *device, GError
 	return fu_plugin_uefi_capsule_write_splash_data (plugin, device, image_bmp, error);
 }
 
+static gboolean
+fu_capsule_update_on_disk (FuDevice *device, GError **error)
```

Comment:
This function should move into `fu_uefi_device_write_firmware`

Reply:
I can move it, but why rename it?  The function only needs to copy the firmware on a special directory, which is what capsule update on-disk is supposed to do for the OS. On he next reboot the firmware will pick it up and do the writing. The _write part can be misleading.

Reply:
Done